tests/unit/test_service.py

```diff
@@ -228,12 +228,42 @@ def _fake_measurement_function():
     pass
 
 
+@pytest.mark.parametrize(
+    "service_config,provided_interfaces",
+    [
+        (
+            "example.serviceconfig",
+            [
+                "ni.measurementlink.measurement.v1.MeasurementService",
+                "ni.measurementlink.measurement.v2.MeasurementService",
+            ],
+        ),
+        ("example.v1.serviceconfig", ["ni.measurementlink.measurement.v1.MeasurementService"]),
+        ("example.v2.serviceconfig", ["ni.measurementlink.measurement.v2.MeasurementService"]),
+    ],
+)
+def test___service_config___create_measurement_service___service_info_matches_service_config(
+    test_assets_directory: str, service_config: str, provided_interfaces: List[str]
+):
+    measurement_service = MeasurementService(
+        service_config_path=test_assets_directory / service_config,
+        version="1.0.0.0",
+        ui_file_paths=[],
+    )
+
+    assert measurement_service.service_info.service_class == "SampleMeasurement_Python"
+    assert set(measurement_service.service_info.provided_interfaces) >= set(provided_interfaces)
+    assert (
+        measurement_service.service_info.description_url
+        == "https://www.example.com/SampleMeasurement.html"
+    )
+
+
 @pytest.fixture
-def measurement_service() -> MeasurementService:
+def measurement_service(test_assets_directory: str) -> MeasurementService:
     """Create a MeasurementService."""
-    assets_directory = pathlib.Path(__file__).resolve().parent.parent / "assets"
     return MeasurementService(
-        service_config_path=assets_directory / "example.serviceconfig",
+        service_config_path=test_assets_directory / "example.serviceconfig",
```

Comment:
This type annotation is wrong. `tests_assets_directory` is a `pathlib.Path`.

Reply:
```
Run poetry run mypy tests
tests/unit/test_service.py:24[9](https://github.com/ni/measurementlink-python/actions/runs/5050063526/jobs/9060299510?pr=274#step:10:10): error: Unsupported left operand type for / ("str")  [operator]
tests/unit/test_service.py:266: error: Unsupported left operand type for / ("str")  [operator]
Found 2 errors in 1 file (checked 15 source files)
Error: Process completed with exit code 1.
```

Reply:
That explains it. :)